Identity

vishvananda opened the pull request:
depends on #32 
implements #9

<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

- New Features
  - Added an “Identity” section with guidance on short‑lived workload credentials, platform-provided validation data, and backing-service verification.

- Documentation
  - Removed the standalone “Admin processes” document and replaced its TOC entry with “Identity.”
  - Consolidated admin/one‑off task guidance into the “Processes” document (new sections on admin tasks and environment parity).
  - Revised configuration guidance formatting and typography without semantic changes.
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

vishvananda committed:
Combine processes factors

vishvananda committed:
first pass at identity

weavenet commented on content/identity.md:
```diff
@@ -0,0 +1,58 @@
+
+- An app retrieves its connection-specific identity from a file like
+  `/var/run/identity/token` referenced by an environment variable (e.g.,
+  `DEFAULT_IDENTITY`).
```
include AWS metadata service as a network example?

weavenet commented on content/identity.md:
```diff
@@ -0,0 +1,58 @@
+  `DEFAULT_IDENTITY`).
+
+##### Guidance
+
```
This focuses on credentials, should we also talk about other aspects of identity 1) identifier (i.e. instance id) 2) policies, what can it access as a result of it's role

vishvananda replied:
i wanted to focus this primarily on creds but i'm not opposed to adding information about that if you want to propose some additions

weavenet commented on content/identity.md:
```diff
@@ -0,0 +1,58 @@
+
+##### Guidance
+
+- Ensure that the workload identity is short-lived to reduce exposure in case of
```
is this referring to credentials?  the identity of the node should last it's lifetime

vishvananda replied:
clarified this guidance to focus on credentials

weavenet commented on content/processes.md:
👍

vishvananda committed:
update and add config changes

vishvananda committed:
Update the identity factor to be more general

Focus on credentials instead of the identity itself, and clarify how
this can be applied to various schemes.

vishvananda committed:
Update identity factor for clarity and to support transaction tokens

This commit refines the language of the identity factor to improve
clarity and explicitly accommodate modern authentication mechanisms
like transaction tokens.

The following changes were made:

- Added transaction tokens as an example of a narrowly scoped,
  short-lived credential.
- Simplified the titles of the first and third principles to be more
  direct and principle-oriented, focusing on the "what" rather than
  the "when" or "how".
- Clarified the descriptions of the first and third principles to
  explicitly cover both connection-based (e.g., mTLS) and
  request-based (e.g., bearer token) authentication models.

vishvananda committed:
update link

vishvananda committed:
fix link

gigako1981 commented:
``
